indexer/map_style.cpp

```diff
@@ -53,3 +53,10 @@ std::string DebugPrint(MapStyle mapStyle)
 {
   return MapStyleToString(mapStyle);
 }
+
+
+bool MapStyleIsDark(MapStyle mapStyle)
+{
+  const std::unordered_set<MapStyle> darkStyles {MapStyle::MapStyleDefaultDark, MapStyle::MapStyleVehicleDark, MapStyle::MapStyleOutdoorsDark};
+  return darkStyles.contains(mapStyle);
+}
```

Comment:
There is no need to pessimize and allocate memory on each call. Looking up in 3 values in a sequential container is way faster than creating a hash and looking into it.
```suggestion
bool IsMapStyleDark(MapStyle mapStyle)
{
  using enum MapStyle;
  for (const auto darkStyle : {MapStyleDefaultDark, MapStyleVehicleDark, MapStyleOutdoorsDark})
  {
    if (mapStyle == darkStyle)
      return true;
  }
  return false;
}
```

Reply:
Changed.